aws_guardduty/lib/botocore/data/storagegateway/2013-06-30/examples-1.json

```diff
@@ -0,0 +1,1381 @@
+{
+  "version": "1.0",
+  "examples": {
+    "ActivateGateway": [
+      {
+        "input": {
+          "ActivationKey": "29AV1-3OFV9-VVIUB-NKT0I-LRO6V",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 4e45642c-4b8c-4052-9ba5-9669f6d62e52 -->